src/CommunityToolkit/Xamarin.CommunityToolkit/Views/RangeSlider/RangeSlider.shared.cs

```
@@ -550,9 +550,11 @@ void OnViewSizeChanged(object sender, System.EventArgs e)
 			OnLayoutPropertyChanged();
 		}
 
-		void OnPanUpdated(object sender, PanUpdatedEventArgs e)
+		void OnPanUpdated(object? sender, PanUpdatedEventArgs e)
 		{
-			var view = (View)sender;
+			if (sender is not View view)
```

Can we do a cast here, and throw an InvalidCastException?